gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -9,11 +9,15 @@ module.exports = function(env) {
   var jsSrc = path.resolve(config.root.src, config.tasks.js.src)
   var jsDest = path.resolve(config.root.dest, config.tasks.js.dest)
   var publicPath = path.join(config.tasks.js.dest, '/')
-  var filenamePattern = env === 'production' ? '[name]-[hash].js' : '[name].js'
+  var filenamePattern = '[name].js'
   var extensions = config.tasks.js.extensions.map(function(extension) {
     return '.' + extension
   })
 
+  if(config.tasks.production.rev.enable === true && env === 'production') {
+    var filenamePattern = '[name]-[hash].js'
```

Comment:
is the "var" here not wrong? (variable filenamePattern has been defined above)
and - why do you use rev.enable instead of just "rev"? do you expect additional config settings there?